CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 
 
 ## Unreleased
+- Updated: Set queue visibility timeout based on lambda timeout
```

Comment:
@daynelucas  maybe call out its an SQS queue here 

Note there be be custom SQS queues created in some of the tenant repos - probably worth doing a grep across all of them and replicating this

Also appreciate the lins below - really good to point at the source so we know why in 6 months time when all is forgotten and have no idea where the recommendation cam from